frontend/src/metabase/nav/components/DatabasePromptBanner/DatabasePromptBanner.unit.spec.tsx

```diff
@@ -69,7 +69,7 @@ async function setup({
   // Then we could safely assert that the banner is not rendered.
   // If we don't wait for this API call to finish, the banner could have rendered,
   // and the test would still pass.
-  if (isAdmin && isPaidPlan) {
+  if (isAdmin && isPaidPlan && !isWhiteLabeling) {
```

Comment:
It's a little different in the actual component vs hook. In the hook tests, we can assert for texts to be **visible** e.g. `hiding xxx` or `showing xxx`, but in these tests, they're either asserting the banner is visible or not at all which is the actual behavior. And we don't have the loading state for this component either. That's why it's a little complicated and cumbersome to test.

Any test that involves testing if _certain things aren't on a screen_ would always be a bit tricky. This is here to ensure that the test is in a state that's ready to assert whether certain elements would **not** be on the screen. e.g. if we don't have this check and assert those elements not being on the screen it would have passed because they're not rendering yet, certain conditions need to be met before those elements will either render or not render at all.